feedthefox/devices/templates/devices.jinja

```diff
@@ -0,0 +1,96 @@
+{% extends "base.jinja" %}
+
+{% block title%} - Devices{% endblock %}
+
+{% block content %}
+  <div class="container">
+
+    <h3>Devices</h3>
+
+    <div class="row">
+      <div class="col-sm-3">
+        <form>
+          <div class="form-group">
+            <label>Manufacturer</label>
+            <select class="form-control" name="manufacturer">
+              <option value="0" selected>All</option>
+              <option value=""></option>
+            </select>
+          </div>
+          <div class="form-group">
+            <label>Type</label>
+            <select class="form-control" name="type">
+              <option value="0" selected>Smartphone</option>
+            </select>
+          </div>
+          <button type="submit" class="btn btn-default">Filter</button>
+        </form>
+      </div>
+      <div class="col-sm-3">
+        <a href="{{ url('device') }}" class="device-list">
+          <div>
+            <img src="" class="img-responsive">
```

Comment:
Yes, it's just a placeholder for each device image. Do you think we should completely remove it for now until we add the tags to fetch devices details from the model?